darwinjail/main.py

```diff
@@ -83,11 +83,25 @@ def copy_files(target_dir: str, queue: dict[str, CopyOpts]) -> None:
         subprocess.check_call(["rsync", "-ah", source_path, full_target_path])
 
         if stat.S_ISREG(st.st_mode):
-            otool_output = (
-                subprocess.check_output(["/usr/bin/otool", "-L", source_path])
-                .decode("UTF-8")
-                .split("\n")
-            )
+            otool_output = ""
+            try:
+                otool_output = (
+                    subprocess.check_output(
+                        ["/usr/bin/otool", "-L", source_path], stderr=subprocess.STDOUT
+                    )
+                    .decode("UTF-8")
+                    .split("\n")
+                )
+            except subprocess.CalledProcessError as err:
+                # Treat this error as warning (do not abort on macOS Monterey)
+                otool_error_datalayout = (
+                    "LLVM ERROR: Sized aggregate specification in datalayout string"
+                )
+                if err.stdout.decode("UTF-8").startswith(otool_error_datalayout):
```

Comment:
I think there should be an `else` branch that would re-raise exception? Otherwise, we're silently swallowing errors.

Reply:
@slonopotamus Fixed.